query/src/test/java/io/keen/client/java/TestKeenQueryClientBuilder.java

```diff
@@ -0,0 +1,24 @@
+package io.keen.client.java;
+
+/**
+ * Created by claireyoung on 5/26/15.
+ */
+public class TestKeenQueryClientBuilder extends KeenQueryClient.QueryBuilder {
```

Comment:
Suggestion: It seems like the only thing this class is doing is overriding the JsonHandler. Maybe you don't need a whole separate class for that? You can always do:

```
new KeenQueryClient.QueryBuilder()
    .withJsonHandler(new TestJsonHandler())
    .build();
```